vendor/vendor.json

```diff
@@ -5379,12 +5369,6 @@
 			"revision": "0f29369cfe4552d0e4bcddc57cc75f4d7e672a33",
 			"revisionTime": "2019-05-07T23:52:07Z"
 		},
-		{
```

Comment:
I wonder where these removals come from? No files were remove in the above.

Reply:
This is strange indeed. let me recheck that.